libs/ui/src/pure/InlineBanner/index.tsx

```diff
@@ -144,17 +184,42 @@ export function InlineBanner({
   children,
   className,
   hideIcon,
-  bannerType,
+  bannerType = 'information',
   borderRadius,
   orientation,
   iconSize,
   iconPadding = '0',
+  customIcon,
   padding,
   margin,
   width,
+  dismissable,
+  bannerID,
 }: InlineBannerProps) {
-  const effectiveBannerType = bannerType || 'alert'
-  const colorEnums = getColorEnums(effectiveBannerType)
+  const colorEnums = getColorEnums(bannerType)
+  const LOCAL_STORAGE_KEY = dismissable ? `${bannerID}` : undefined
+
+  const [showBanner, setShowBanner] = useState(() => {
+    if (dismissable && LOCAL_STORAGE_KEY) {
+      const storedVisibility = localStorage.getItem(LOCAL_STORAGE_KEY)
+      return storedVisibility !== 'false'
+    }
+    return true
+  })
```

Comment:
I personally would prefer to invert the logic.

Show the banner by default.
The flag would be to determine whether it should be hidden.
Thus, we don't need a localStorage value to decide whether to show. If it's set, though, then we hide it.

If you don't mind I can give it a try in a follow up PR.

Reply:
I understand. From what I noticed in the app is that, if we do the inverse (show by default and hide after a certain state/localstorage setting) it causes sometimes to briefly 'flash' these items (race condition, slight delay, etc.). Feel free to give it a go otherwise.

Reply:
Got it!
Ok, let's keep it that way then